src/model/games/export/by_user.rs

```diff
@@ -39,8 +39,8 @@ pub enum Sort {
 pub type GetRequest = crate::model::Request<GetQuery>;
 
 impl GetRequest {
-    pub fn new(game_id: &str, query: GetQuery) -> Self {
-        let path = format!("/api/games/user/{}", game_id);
+    pub fn new(user_id: &str, query: GetQuery) -> Self {
+        let path = format!("/api/games/user/{}", user_id);
```

Comment:
```suggestion
    pub fn new(username: &str, query: GetQuery) -> Self {
        let path = format!("/api/games/user/{}", username);
```
Thanks for spotting this. Looks like this should be username according to the docs though. https://lichess.org/api#tag/Games/operation/apiGamesUser